src/Physics/Resonance/XSection/BostedChristyEMPXSec.cxx

```diff
@@ -0,0 +1,1173 @@
+
+    double F1M = MEC2009(A, Q2, W);
+
+    W1 += F1M;
+    if(W2>0.) 
+      R *= (1.110952 + 4.928E-03*A);
+    
+    
+  }
+  bool goodfit;
+  double emcfac = FitEMC(x, A, goodfit);
+
+  double F1 = MN*W1*emcfac;  
+    
+  double nu2 = nu*nu;
+  double K = (W2 - MN2)/2./MN;
+  double Eprime = E - nu;
+  double sin2theta_2 = Q2/4/E/Eprime;
+  double cos2theta_2 = 1 - sin2theta_2;
+  double tan2theta_2 = sin2theta_2/cos2theta_2;
+  double eps = 1./(1. + 2*(1+nu2/Q2)*tan2theta_2);              // Ref.1, Eq. (4)
+  double Gamma = kAem*Eprime*K/Q2/E/(1-eps)/2/kPi2;             // Ref.1, Eq. (5)
+  sigmaT = 4*kPi2*kAem*F1/K/MN;
+  sigmaL = R*sigmaT;
+  double xsec = Gamma*(sigmaT+eps*sigmaL);                      // Ref.1, Eq. (3) d2xsec/dOmegadEprime
+  double jacobian = W*kPi/E/Eprime/MN;  
+  xsec*= jacobian;                                              // d2xsec/dOmegadEprime-> d2xsec/dWdQ2
+  
+  // The algorithm computes d^2xsec/dWdQ2
+  // Check whether variable tranformation is needed
+  if(kps!=kPSWQ2fE) {
+    double J = utils::kinematics::Jacobian(interaction,kPSWQ2fE,kps);
+    xsec *= J;
+  }
+  
+  return xsec;                              
+  
+}
+//____________________________________________________________________________
+// Fit to EMC effect.  Steve Rock 8/3/94                                 
+// Funciton returns value of sigma(A)/sigma(d) 
+// with no isoscalerity correction
+// A = atomic number                                                      
+// x = Bjorken x.                                                        
+//                                                                       
+// Fit of sigma(A)/sigma(d) to form C*A**alpha where A is atomic number  
+// First data at each x was fit to form C*A**alpha.  The point A=2(d)    
+//  was includded with a value of 1 and an error of about 2%.             
+// For x>=.125 Javier Gomez fit of 7/93 to E139 data was used.           
+// For .09 >=x>=.0085 NMC data from Amaudruz et al Z. Phys C. 51,387(91) 
+//  Steve did the fit for alpha and C to the He/d. C/d and Ca/d NMC data.
+// Alpha(x) was fit to a 9 term polynomial a0 +a1*x +a2*x**2 + a3*x**3 ..
+// C(x) was fit to a 3 term polynomial in natural logs as                
+//  Ln(C) = c0 + c1*Ln(x) + c2*[Ln(x)]**2.                               
+//
+// 6/2/98 *****  Bug (which set x= .00885 if x was out of range) fixed
+//                    also gave value at x=.0085 if x>.88
+// 11/05 PYB PEB modified to use value at x=0.7 if x>0.7, because
+//    beyond that assume Fermi motion is giving the rise, and we
+//    already are taking that into account with the y-smearing of
+//    the inelastic
+//____________________________________________________________________________
+double BostedChristyEMPXSec::FitEMC(double x, int A, bool & goodfit) const
+{                                                                   
+  std::array<double, 9> alpha_coef                                  
```

Comment:
Same here. Comments and static constexpr.

Reply:
Done.